### PR TITLE
added autoupdater for humane-js

### PR DESCRIPTION
**Pull request by seun40**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Should not bump the version here.

**Comment by seun40** [comment]
version reverted

**Comment by PeterDaveHello** [comment]
Please squash them into only one commit.

File: ajax/libs/humane-js/package.json [hunk 1]
```diff
@@ -19,5 +19,20 @@
...
       "url": "https://github.com/wavded/humane-js"
     }
-  ]
+  ],
+    "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wavded/humane-js.git",
+    "basePath": "",
+    "files": [
+      "humane-min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/humane-js/package.json]: `humane.min.js` not `humane-min.js`

File: ajax/libs/humane-js/package.json [hunk 2]
```diff
@@ -19,5 +19,20 @@
...
+      "humane.js",
+      "themes/bigbox.css",
+      "themes/boldlight.css",
+      "themes/flatty.css",
+      "themes/jackedup.css",
+      "themes/libnotify.css",
+      "themes/original.css"
```
Comment (PeterDaveHello) [inline ajax/libs/humane-js/package.json]: Use `themes/*.css` will be better.

**Comment by seun40** [comment]
Peter, can I rebase without a local copy of the fork?

**Comment by PeterDaveHello** [comment]
Maybe not, if you think the process is slow, you should try sparse-checkout.

**Pull request closed without merging**